certified-connectors/KyndrylMainframe/apiDefinition.swagger.json

```diff
@@ -0,0 +1,707 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "Kyndryl mainframe connector",
+    "version": "1.0.0",
+    "title": "kyndryl"
+  },
+  "host": "kyndryl-apigateway.azure-api.net",
+  "basePath": "/",
+  "schemes": [
+    "http"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/cb12-policy/getpolicy": {
+      "get": {
+        "tags": [
+          "cb12-policy"
+        ],
+        "operationId": "getPolicy",
```

Comment:
@RTkyndryl all operationIds should be in Camel Case. please change to "GetPolicy". Please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations). 
Please fix all operationIds in the swagger file.